### PR TITLE
Update install script for husky v9

### PR DESCRIPTION
**Pull request by ohaddahan**
## Description

Changes according to https://github.com/typicode/husky/releases/tag/v9.0.1

## Checklist

- [X] I’ve added tests to confirm my change works.
- [X] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [X] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [X] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -141,9 +139,8 @@ For example, you can do the following to have Prettier run before each commit:
 
    ```bash
    pnpm add --save-dev husky lint-staged
-   pnpm exec husky install
-   npm pkg set scripts.prepare="husky install"
-   pnpm exec husky add .husky/pre-commit "pnpm exec lint-staged"
+   npx husky init
+   echo "npx lint-staged" > .husky/pre-commit
```
Comment (fisker) [inline docs/install.md:143]:
```suggestion
   echo "pnpm exec lint-staged" > .husky/pre-commit
```

File: docs/install.md [hunk 2]
```diff
@@ -121,18 +121,16 @@ For example, you can do the following to have Prettier run before each commit:
 
    ```bash
    npm install --save-dev husky lint-staged
-   npx husky install
-   npm pkg set scripts.prepare="husky install"
-   npx husky add .husky/pre-commit "npx lint-staged"
+   npx husky init
+   echo "npx lint-staged" > .husky/pre-commit
```
Comment (fisker) [inline docs/install.md:125]: I forgot this doesn't work on Windows...

Let's use 

```js
node --eval "fs.writeFileSync('.husky/pre-commit','npx lint-staged')"
```

See https://github.com/prettier/prettier/blob/c8ba8dbca18858a7962184bbb3898502b9ec7cfb/docs/install.md?plain=1#L31-L36

Comment (ohaddahan) [reply docs/install.md]: Updated

**Review by fisker — APPROVED** [review]

**Comment by fisker** [comment]
Would you like to remind husky to update their script?

**Comment by ohaddahan** [comment]
> Would you like to remind husky to update their recommended scripts?

Which part? The `windows` compatibility?

**Comment by fisker** [comment]
> The windows compatibility?

Yes.

**Comment by ohaddahan** [comment]
> > The windows compatibility?
> 
> Yes.

Pretty sure their assumption is `git bash` is used, they specifically mention `On windows`.

**Comment by fisker** [comment]
https://github.com/typicode/husky/issues/1380

**Pull request merged**